### ISSUE TITLE
How to allow Windows Update?

### ISSUE THREAD

**Issue by adolfintel** [henrypp/simplewall#370]
Allowing wuauserv doesn't allow Windows to search for updates, and neither does the builtin rule for Windows Update in the settings.
Whitelisting svchost.exe allows it but it's definitely overkill.

Any suggestions? This is on 1809 and 19H1

**Comment by Iruberiam** [comment]
Also on 1809, enabled Windows Updates in Simplewalls settings but still had to set a timer for svchost to check for updates.

**Comment by henrypp** [comment]
- [How to fix Windows Update and Windows Store internet access (temporary solution)](https://github.com/henrypp/simplewall/issues/206#issuecomment-439830634)

**Comment by henrypp** [comment]
3.0.1 can allow Microsoft apps/update servers in-built!

**Comment by adolfintel** [comment]
Nice 👍

**Comment by Iruberiam** [comment]
Thanks, the version 3.1 beta is great.

**Comment by odd-look** [comment]
Hi. I'm new to GitHub and I hope commenting here is correct behavior. 
I'm using Simplewall for a while now but since I block svchost.exe, Windows updates are not possible.

I'm waiting for an fix since a fairly long time now. I tried the "temporary solution" approach from Jun 2018 and a few days ago I tried 3.0.5 BETA. I had hope that updates would work in v3.x but it doesn't for me.

The only working "solution" was to allow svchost.exe but that's not really a solution...

I don't understand why this issue is marked as closed?
Am I the only one who still has this problem?
Is it working for you in 3.x?

Is a solution realistic in the near future?
